src/main/java/Ui.java

```diff
@@ -0,0 +1,130 @@
+import java.util.Scanner;
+
+public class Ui {
+    Scanner scanner;
+    private boolean isExit;
+    private Parser parser;
+
+    public Ui() {
+        scanner = new Scanner(System.in);
+        isExit = false;
+        parser = new Parser();
+    }
+
+    public void showLine() {
+        System.out.println("    ____________________________________________________________");
+    }
+
+    public void showWelcome() {
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        System.out.println("Hello from\n" + logo);
+        showLine();
+        System.out.println("     Hello! I'm Duke\n     What can I do for you?");
+        showLine();
+        System.out.println();
+    }
+
+    public void execute(TaskList tasks) {
```

Comment:
I like the naming of the parameter : tasks, as it suggest it is a list, not one item